builds/any/rootfs/buster/common/arm64-base-packages.yml

```
@@ -1 +1,15 @@
 - u-boot-tools
+- onl-kernel-5.15-lts-arm64-all-modules
+- frr
+- mrvl-fw-image
+- bridge-utils
+- bash-completion
+- man-db
+- dnsmasq
+- dnsutils
+- keepalived
+- lldpd
+- mstpd
+- hostapd
+- stress-ng
+- python3-pyroute2
```

to keep the order of the stretch version:
```suggestion
- frr
- bridge-utils
- mstpd
- bash-completion
- onl-kernel-5.15-lts-arm64-all-modules
- lldpd
- man-db
- dnsmasq
- dnsutils
- mrvl-fw-image
- keepalived
- hostapd
- stress-ng
- python3-pyroute2
```

you are dropping mfr, intentional?